osu.Game.Rulesets.Catch/Objects/Drawables/DrawableCatchHitObject.cs

```diff
@@ -70,6 +70,8 @@ public abstract class DrawableCatchHitObject : DrawableHitObject<CatchHitObject>
 
         public float DisplayRadius => DrawSize.X / 2 * Scale.X * HitObject.Scale;
 
+        protected override float SamplePlaybackBalance => 0.8f * HitObject.X - 0.4f;
```

Comment:
where does 0.8/0.4 come from here?

Reply:
right, i see where you're coming from. going to change the 0.8f into a shared constant for legibility.